docs/Getting-Started.md

```diff
@@ -21,6 +21,13 @@ yarn add fastify
 Let's write our first server:
 ```js
 // Require the framework and instantiate it
+
+// ESM
```

Comment:
Just to share, would it be more straightforward for users get a table

|esm|cjs|
|---|---|
code | code|

?

Reply:
Won't this break the doc website? I will be hard to read on all devices

Reply:
Just for curiosity, how the website would looks like with:

<details>
  <summary>ESM</summary>
  
  ```js
  import Fastify from 'fastify'
  const fastify = Fastify({
    logger: true
  })
```
</details>

<details>
  <summary>Commonjs</summary>
  
  ```js
  const fastify = require('fastify')({
    logger: true
  })
```
</details>

?